specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -3852,6 +3852,11 @@
           "description": "Indicates whether user has read access to the billing account.",
           "type": "boolean",
           "readOnly": true
+        },
+        "isTransitioned": {
+          "description": "Specifies if the billing account has transitioned from agreement type Microsoft Online Service Program to agreement type Microsoft Customer Agreement.",
```

Comment:
According to version guidelines this is considered a breaking change, so I doubt we can add it to the stable version.
See: [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning)